numpyro/distributions/continuous.py

```diff
@@ -708,6 +715,8 @@ def __init__(self, loc=0., covariance_matrix=None, precision_matrix=None, scale_
                  validate_args=None):
         if jnp.ndim(loc) == 0:
             loc, = promote_shapes(loc, shape=(1,))
+        # FIXME this expansion can lead to expensive gradient computation; for
+        # discussion see https://github.com/pytorch/pytorch/issues/43837
```

Comment:
In NumPyro, we don't expand parameters (to save some computation in logprob) and only add singleton dimensions to the parameters.

Reply:
Thanks for explaining! removed.